commands/azureCommands/delete-image.ts

```diff
@@ -14,34 +14,64 @@ import { Repository } from "../../utils/Azure/models/repository";
 import { getLoginServer } from "../../utils/nonNull";
 import * as quickPicks from '../utils/quick-pick-azure';
 
```

Comment:
Could you change the wizard wording 'Select registry to use' to simply 'Select registry'?  It feels weird for things like untagging.  Thx.

export async function quickPickACRRegistry(canCreateNew: boolean = false, prompt?: string): Promise<Registry> {
    const placeHolder = prompt ? prompt : 'Select registry to use';

Reply:
done